### PR TITLE
Feature/transform persist

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**What change does this PR introduce?** 
 
added a `transform` argument to the transform package passthrough column macro, so that we could aggregate them

**If this PR introduces a new macro, how did you test the new macro?** 

tested in Pendo and other packages 

**If this PR introduces a modification to an existing macro, which packages is the macro currently present in and what steps were taken to test compatibility across packages?** 

pendo, twitter_ads 

**Did you update the README to reflect the macro addition/modifications?** 

- [x] Yes
- [ ] No (provide further explanation)

### CODE REVIEW

File: macros/persist_pass_through_columns.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
-{% macro persist_pass_through_columns(pass_through_variable) %}
+{% macro persist_pass_through_columns(pass_through_variable, identifier, transform='') %}
```
Comment (fivetran-joemarkiewicz) [inline macros/persist_pass_through_columns.sql:1]: This works, but I would feel more comfortable if we added a default value for `identifier`, just to be safe.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Just left the one comment to add a default value for `identifier`. Once that is applied we will be good to go!

I do want to hold off merging until Monday just so we are prepared in case anything comes up.

File: macros/persist_pass_through_columns.sql (hunk 1 above)
Comment (fivetran-jamie) [reply macros/persist_pass_through_columns.sql]: added identifier=none

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-jamie I have tested this some more and it looks good to go!

Before we cut this release would you mind updating the following:
- [x] Remove the .DS_Store file from the PR.
- [x] Add a CHANGELOG entry.

**Comment by fivetran-jamie** [comment]
@fivetran-joemarkiewicz updates are made!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**